poppy/tests/coveragerc

```diff
@@ -1,16 +1,16 @@
 [run]
-source = {packagename}
+source = poppy
 omit =
-   {packagename}/_astropy_init*
-   {packagename}/conftest*
-   {packagename}/cython_version*
-   {packagename}/setup_package*
-   {packagename}/*/setup_package*
-   {packagename}/*/*/setup_package*
-   {packagename}/tests/*
-   {packagename}/*/tests/*
-   {packagename}/*/*/tests/*
-   {packagename}/version*
+   poppy/_astropy_init*
+   poppy/conftest*
+   poppy/cython_version*
+   poppy/setup_package*
+   poppy/*/setup_package*
+   poppy/*/*/setup_package*
+   poppy/tests/*
+   poppy/*/tests/*
+   poppy/*/*/tests/*
+   poppy/version*
```

Comment:
If you use `tox`, you also need to repeat these lines with `*/` in front, according to Tom R.

Reply:
Also in the "final" guide published in http://docs.astropy.org/projects/package-template/en/latest/ape17.html#step-15-coverage-configuration , this file should be removed altogether and the config here moved to `setup.cfg`.